### ISSUE TITLE
It doesn't call the function

### ISSUE THREAD

**Issue by tonylampada** [MeetKai/functionary#53]
I'm running it with this command

`python3 server.py --model "meetkai/functionary-7b-v1.1"`

then I'm running your chatlab example, and I get this output:

```
USER: what is the price of the car named Tang?
ASSISTANT:  get_car_price:
{"car_name": "Tang"}
```

that is, it gets back a json with `content` instead of `function_call`

Am I doing anything wrong?

**Comment by zayd-meetkai** [comment]
Hi, 

server.py is actually just a vanilla server that does not parse function calls and does not handle the required special tokens, hence the result. We have removed it for now. Please use the VLLM server as such:

`python3 server_vllm.py --model "meetkai/functionary-7b-v1.1" --host 0.0.0.0`

Also, for the chatlab example, please use chatlab==0.16.0 since the current releases don't support customizing the api_base, and a have a completely different format.

Thank you!

**Comment by rgbkrk** [comment]
> Also, for the chatlab example, please use chatlab==0.16.0 since the current releases don't support customizing the api_base, and a have a completely different format.

I recently added custom server support for chatlab! https://github.com/rgbkrk/chatlab/pull/115

I'll make sure to get a release out tonight with it.

**Comment by rgbkrk** [comment]
Released chatlab 1.1.1 with custom server support.

https://github.com/rgbkrk/chatlab/releases/tag/v1.1.1

```python
from chatlab import Chat

chat = Chat(
  base_url="http://localhost:8000/v1",
  api_key="functionary"
)
```

Technically it supported it before. However, the new `openai` bindings swapped the name to `base_url`. I tried to make this easier to work with by passing it into the `Chat()` constructor.
